extension.driver.php

```diff
@@ -24,7 +24,7 @@ public function install()
 					`destination` VARCHAR(255) NOT NULL,
 					`validator` VARCHAR(50),
 					`unique` enum('yes','no') NOT NULL DEFAULT 'yes',
-					`default_main_lang` ENUM('yes', 'no') CHARACTER SET utf8 COLLATE utf8_unicode_ci NOT NULL DEFAULT 'no',
+					`default_main_lang` ENUM('yes', 'no', '') CHARACTER SET utf8 COLLATE utf8_unicode_ci NOT NULL DEFAULT 'no',
```

Comment:
Because without it we get this error : 

![whyenumneedempty](https://cloud.githubusercontent.com/assets/8250836/17708458/1082427c-63b3-11e6-96d1-caee04cba8ce.PNG)






[config.txt](https://github.com/DeuxHuitHuit/multilingual_image_upload/files/420862/config.txt)
[debugdevkit-master.zip](https://github.com/DeuxHuitHuit/multilingual_image_upload/files/420863/debugdevkit-master.zip)